diff --git a/src/liballoc/lib.rs b/src/liballoc/lib.rs
index 63b3fbb..977ef16 100644
--- a/src/liballoc/lib.rs
+++ b/src/liballoc/lib.rs
@@ -1,3 +1,9 @@
+//! **You are viewing the documentation of the Rust core allocation and
+//! collections library as a part of the Fortanix Rust EDP.** There are no
+//! changes compared to the normal the core allocation and collections library.
+//!
+//! ---
+//!
 //! # The Rust core allocation and collections library
 //!
 //! This library provides smart pointers and collections for managing
@@ -52,7 +58,9 @@

 #![allow(unused_attributes)]
 #![stable(feature = "alloc", since = "1.36.0")]
-#![doc(html_root_url = "https://doc.rust-lang.org/nightly/",
+#![doc(html_logo_url = "https://edp.fortanix.com/img/docs/edp-logo.svg",
+       html_favicon_url = "https://edp.fortanix.com/favicon.ico",
+       html_root_url = "https://edp.fortanix.com/docs/api/",
        issue_tracker_base_url = "https://github.com/rust-lang/rust/issues/",
        test(no_crate_inject, attr(allow(unused_variables), deny(warnings))))]
 #![no_std]
diff --git a/src/libcore/lib.rs b/src/libcore/lib.rs
index 63688e7..881e8f1 100644
--- a/src/libcore/lib.rs
+++ b/src/libcore/lib.rs
@@ -1,3 +1,9 @@
+//! **You are viewing the documentation of the Rust core library as a part of
+//! the Fortanix Rust EDP.** There are no changes compared to the normal the
+//! core library.
+//!
+//! ---
+//!
 //! # The Rust Core Library
 //!
 //! The Rust Core Library is the dependency-free[^free] foundation of [The
@@ -51,7 +57,9 @@
 #![cfg(not(test))]

 #![stable(feature = "core", since = "1.6.0")]
-#![doc(html_root_url = "https://doc.rust-lang.org/nightly/",
+#![doc(html_logo_url = "https://edp.fortanix.com/img/docs/edp-logo.svg",
+       html_favicon_url = "https://edp.fortanix.com/favicon.ico",
+       html_root_url = "https://edp.fortanix.com/docs/api/",
        html_playground_url = "https://play.rust-lang.org/",
        issue_tracker_base_url = "https://github.com/rust-lang/rust/issues/",
        test(no_crate_inject, attr(deny(warnings))),
diff --git a/src/libstd/lib.rs b/src/libstd/lib.rs
index d11dee8..12d82d0 100644
--- a/src/libstd/lib.rs
+++ b/src/libstd/lib.rs
@@ -1,3 +1,11 @@
+//! **You are viewing the documentation of the Rust standard library as a part
+//! of the Fortanix Rust EDP.** The only changes compared to the normal Rust
+//! standard library are the addition of the [`std::os::fortanix_sgx` module].
+//!
+//! [`std::os::fortanix_sgx` module]: os/fortanix_sgx/
+//!
+//! ---
+//!
 //! # The Rust Standard Library
 //!
 //! The Rust Standard Library is the foundation of portable Rust software, a
@@ -196,7 +204,9 @@
 //! [primitive types]: ../book/ch03-02-data-types.html

 #![stable(feature = "rust1", since = "1.0.0")]
-#![doc(html_root_url = "https://doc.rust-lang.org/nightly/",
+#![doc(html_logo_url = "https://edp.fortanix.com/img/docs/edp-logo.svg",
+       html_favicon_url = "https://edp.fortanix.com/favicon.ico",
+       html_root_url = "https://edp.fortanix.com/docs/api/",
        html_playground_url = "https://play.rust-lang.org/",
        issue_tracker_base_url = "https://github.com/rust-lang/rust/issues/",
        test(no_crate_inject, attr(deny(warnings))),
